tests/test_lightning_classification_pipeline.py

```diff
@@ -16,18 +17,29 @@ def pipeline_kwargs() -> Dict[str, Any]:
 
 
 @pytest.fixture(scope="module")
-def dataset_kwargs() -> Dict[str, Any]:
-    return {
-        "dataset_name_or_path": "clarin-pl/polemo2-official",
-        "input_column_name": ["text"],
-        "target_column_name": "target",
-        "load_dataset_kwargs": {
+def dataset_kwargs() -> Tuple[Dict[str, Any], "TemporaryDirectory[str]"]:
+    path = TemporaryDirectory()
+    pipeline = HuggingFacePreprocessingPipeline(
+        dataset_name="clarin-pl/polemo2-official",
+        load_dataset_kwargs={
             "train_domains": ["hotels", "medicine"],
             "dev_domains": ["hotels", "medicine"],
             "test_domains": ["hotels", "medicine"],
             "text_cfg": "text",
         },
-    }
+        persist_path=path.name,
+        sample_missing_splits=None,
+        ignore_test_subset=False,
+        downsample_splits=(0.01, 0.01, 0.05),
+        seed=441,
+    )
+    pipeline.run()
+
+    return {
+        "dataset_name_or_path": path.name,
+        "input_column_name": ["text"],
+        "target_column_name": "target",
+    }, path
```

Comment:
do we need the path to be returned if later in the pipeline we use `dataset_kwargs[0]`?

Reply:
Otherwise the temporary path is destroyed. Passing `path.name` is not sufficient to keep object